lara-typescript/src/interactive-api-client/types.ts

```diff
@@ -297,6 +299,12 @@ export interface ICloseModal {
   uuid?: string;
 }
 
+export interface IDecoratedContentEventMessage {
```

Comment:
Based on other Message Interfaces, the `Message` suffix should be dropped from this interface name. The name of interface seems to match the name of the message itself.  Unless it is a message which requires a response then it seems to be have Request and Response suffixes.

Reply:
Got it.  Will change.

Reply:
Fixed.